Makefile

```diff
@@ -38,6 +38,7 @@ TEST_E2E_DIR := test/e2e
 
 # Binaries.
 CLUSTERCTL := $(BIN_DIR)/clusterctl
+KUSTOMIZE := $(abspath $(TOOLS_BIN_DIR)/kustomize)
```

Comment:
macOS doesn't have `abspath` out of the box. I only see us using `abspath` for `GINKGO`. I wonder if we can remove it, or, if we do need an absolute path, we can do `cd <path> && pwd`.